lua/entities/gmod_wire_expression2/core/custom/cfc_miscellaneous.lua

```diff
@@ -0,0 +1,5 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
```

Comment:
We should have a file dedicated to constants if that's what we want to do. It should be scoped for `/custom/constants/cfc_contants.lua` or /custom/cfc_constants.lua`